### PR TITLE
Fix `make docs`

### LINKED ISSUE #194: ``make docs`` broken
jensens@minime:~/workspace/plone.api$ make docs
    bin/buildout 
    Develop: '/home/jensens/workspace/plone.api/.'
    [...]
    Install Git pre-commit hook.
    bin/sphinx-build -W docs docs/html
    Running Sphinx v1.1.3
    loading translations [en]... locale not available
    loading pickled environment... done
    No builder selected, using default: html
    building [html]: targets for 19 source files that are out of date
    updating environment: [config changed] 19 added, 0 changed, 0 removed
    reading sources... [100%] user                                                                                                                    
    looking for now-outdated files... none found
    pickling environment... done
    checking consistency... 
    Warning, treated as error:
    /home/jensens/workspace/plone.api/docs/CHANGES.rst:: WARNING: document isn't included in any toctree

    make: *** [docs/html/index.html] Error 1

### PR DESCRIPTION
**Pull request by jaroel**
Fixes #194

### CODE REVIEW

File: docs/CHANGES.rst [hunk 1]
```diff
@@ -3,6 +3,8 @@ Changelog
 
 1.3.3 (unreleased)
 ------------------
+- Fixed `make docs`.
```
Comment (zupo) [inline docs/CHANGES.rst]: There should be one empty line between header and the first item.

**Comment by zupo** [comment]
One minor styling nitpick, otherwise looks good!

**Comment by jaroel** [comment]
Please review.

**Comment by zupo** [comment]
Perfect, thanks!

**Pull request merged**